styles/style.css

```diff
@@ -0,0 +1,127 @@
+#nav {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 50px;
+}
+
+#nav a {
+  text-decoration: none;
+  font-family: Poppins, sans-serif;
+  font-weight: bold;
+  font-size: 30px;
+  color: #1e31d8;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  margin-left: 24px;
+}
+
+#nav a:hover {
+  text-decoration: underline;
+  text-decoration-color: #23a3fa;
+  text-decoration-thickness: 2px;
+}
+
+#nav nav {
+  width: 45px;
+  height: 30px;
+  display: flex;
+  align-items: center;
+}
+
+section {
+  background-image:
+    url("../images/Ellipse 11.png"),
+    url("../images/Rectangle 4.png"),
+    url("../images/Ellipse\ 3.png"),
+    url("../images/Ellipse\ 8.png"),
+    url("../images/Ellipse\ 5.png"),
+    url("../images/Rectangle\ 7.png"),
+    url("../images/Ellipse 12.png"),
+    url("../images/Ellipse 4.png"),
+    url("../images/Rectangle\ 5.png"),
+    url("../images/Ellipse\ 13.png"),
+    url("../images/Ellipse\ 15.png");
+  background-position:
+    19px 79px,
+    294px 68px,
+    323px 102px,
+    214px 110px,
+    125px 106px,
+    127px 98px,
+    370px 385px,
+    290px 509px,
+    240px 507px,
+    357px 392px,
+    106px 497px;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 600px;
+}
+
+section h1 {
+  font-family: Poppins, sans-serif;
+  font-weight: bold;
+  font-size: 40px;
+  color: #172b4d;
+  line-height: 52px;
+  text-align: start;
+  margin: 0 24px;
+}
```

Comment:
- [ ] You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Kindly Adjust the background image alignment just like design and adjust font-weight of main title("I'm Itumeleng. So glad to see you!") and "Let's connect " Just Like figma design.

Your Project Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/199048000-37a12f6b-dd02-4440-a834-2f88d5372430.PNG)

The Figma design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/199048289-cde119e4-0bf9-4fc6-b832-a884b777478a.PNG)